spec/marten/cli/manage/command/collect_assets_spec.cr

```diff
@@ -1,4 +1,6 @@
 require "./spec_helper"
+require "digest/sha256"
+require "json"
```

Comment:
We probably don't need to require `json` here as it should already be required by Marten (other JSON operations are already performed by the framework!).